src/exams/exams.controller.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  UseGuards,
+  UseInterceptors,
+} from "@nestjs/common";
+
+import { Roles } from "@/auth/decorators/roles.decorator";
+import { JwtAuthGuard } from "@/auth/guards/jwt-auth.guard";
+import { RolesGuard } from "@/auth/guards/roles.guard";
+import { EUserRole } from "@/common/enums/roles.enum";
+import { ResponseTransformInterceptor } from "@/common/interceptors/response-transform.interceptor";
+
+import { CreateExamDto, UpdateExamDto } from "./exams.dto";
+import { ExamsService } from "./exams.service";
+
+@UseInterceptors(ResponseTransformInterceptor)
+@UseGuards(JwtAuthGuard)
+@Controller("classrooms")
+export class ExamsController {
+  constructor(private readonly examsService: ExamsService) {}
+
+  @Get(":id/exams")
```

Comment:
```suggestion
  @Get(":classroomId/exams")
```

Let's be more specific about which id this is. Same comment wherever applicable throughout the PR